modules/conversantAnalyticsAdapter.js

```diff
@@ -0,0 +1,544 @@
+import {ajax} from '../src/ajax.js';
+import adapter from '../src/AnalyticsAdapter.js';
+import CONSTANTS from '../src/constants.json';
+import {getGlobal} from '../src/prebidGlobal.js';
+import adapterManager from '../src/adapterManager.js';
+import * as utils from '../src/utils.js';
+
+const {
+  EVENTS: { AUCTION_END, AD_RENDER_FAILED, BID_TIMEOUT, BID_WON }
+} = CONSTANTS;
+const GVLID = 24;
+const ANALYTICS_TYPE = 'endpoint';
+
+// for local testing set domain to 127.0.0.1:8290
+const URL = 'https://web.hb.ad.cpe.dotomi.com/cvx/event/prebidanalytics';
+const ANALYTICS_CODE = 'conversant';
+
+export const CNVR_CONSTANTS = {
+  LOG_PREFIX: 'Conversant analytics adapter: ',
+  // Maximum time to keep an item in the cache before it gets purged
+  MAX_MILLISECONDS_IN_CACHE: 30000,
+  // How often cache cleanup will run
+  CACHE_CLEANUP_TIME_IN_MILLIS: 30000,
+  // Should be float from 0-1, 0 is turned off, 1 is sample every instance
+  DEFAULT_SAMPLE_RATE: 1,
+
+  // BID STATUS CODES
+  WIN: 10,
+  BID: 20,
+  NO_BID: 30,
+  TIMEOUT: 40,
+  RENDER_FAILED: 50
+};
+
+// Saves passed in options from the bid adapter
+let initOptions;
```

Comment:
This should be a constant object rather than be assigned to the global config object.
```javascript
const initOptions = {};
```